contracts/anchor_basset_hub/src/testing/tests.rs

```diff
@@ -1856,8 +1858,7 @@ pub fn proper_slashing() {
     let res = execute(deps.as_mut(), mock_env(), info.clone(), second_bond).unwrap();
     assert_eq!(2, res.messages.len());
 
-    // expected exchange rate must be more than 0.9
-    let expected_er = Decimal::from_ratio(Uint128::from(1900u64), Uint128::from(2111u64));
+    // expected exchange rate must be equal 0.9
```

Comment:
"equal", are you sure? er was 0.9 and after a second bond(1858) it should be a little bigger.

Reply:
Fixed